app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split(" ")
+
+    if len(command_parts) != 3 or command_parts[0] != "cp":
+        raise ValueError(
+            "Invalid command format. "
+            "Expected format: 'cp <source_file> <destination_file>'"
```

Comment:
The checklist suggests using a consistent style of quotes, preferably double quotes. Consider changing the single quotes to double quotes here for consistency.